aten/src/ATen/native/Vision.cpp

```diff
@@ -0,0 +1,23 @@
+#include "ATen/ATen.h"
+#include "ATen/NativeFunctions.h"
+
+#define MODE_ZEROS 0
+#define MODE_BORDER 1
+
+namespace at { namespace native {
+
+Tensor grid_sampler(const Tensor& input, const Tensor& grid, int64_t padding_mode) {
+  // as of cudnn 7102, will not work for larger than 1024
```

Comment:
Do we support cudnn < 7102? If so, we should guard this using ifdefs

Reply:
we definitely should support cudnn 7

Reply:
Also, is this a cudnn bug or feature? If it is not intentional then maybe we should print a warning

Reply:
Not sure if it's intended behavior, I will have to check on that.